docs/editors/online-ides.md

```diff
@@ -0,0 +1,66 @@
+---
+id: online-ides
+title: Online IDEs
+---
+
+Metals can also be installed in some online environments, which enable users to
+work on their code inside of a browser. This can be good option when users need
+to setup their workspace quickly or are lacking appropriate resources on their
+local machine.
+
+# Gitpod
+
+One of such online environments is Gitpod, which can be used directly in your
+browser as well as shared with others without a need to explicitly set up the
+tools and settings needed to run Scala project with Metals. You can read more
+about Gitpod [here](https://www.gitpod.io/docs/).
+
+## Example Gitpod repository for Metals
+
+An example repository, which is already set up and integrated with Gitpod can be
+found [here](https://github.com/scalameta/metals-gitpod-sample). The project
+contains, out of the box, everything that you need to start experimenting with
+Metals in an online environment similar to Visual Studio Code. The only thing
+that you need to do to set up the environment by yourself is to open the
+[link](https://gitpod.io/#https://github.com/scalameta/gitpod-sample) provided
+in the README of the project. The import prompt for Metals will show up
+(however, in general, it wouldn't be needed if the _.bloop_ files exist).
```

Comment:
I think we should maybe clarify what we mean here a bit. Since this isn't always fully true, and readers may interpret this as any time there is `.bloop` files, then we are good. However, there may be cases where you still need an import with existing `.bloop` files.

Reply:
Reworded this a bit to mention that it's only the case in this setup